recipes.js

```diff
@@ -1,11 +1,50 @@
 const mongoose = require('mongoose');
+
+const Recipe = require("./recipe-model.js");
+
 const Schema   = mongoose.Schema;
 const data = require('./data.js')
 
+
+
 mongoose.connect('mongodb://localhost/recipeApp')
   .then(() => {
     console.log('Connected to Mongo!')
   }).catch(err => {
     console.error('Error connecting to mongo', err)
   });
 
+  // Recipe.create({title, level, ingredients, cousine, dishType, image, duration, creator, created})
+  // .then(recipeDoc => {
+  //   console.log("recipe create SUCCESS!!", recipeDoc)
+  // })
+  // .catch(err => {
+  //   console.log("recipe create FAILURE!! 💩", err);
+  // });
+
+Recipe.insertMany(data)
+.then((oneRecipe) => {
+  console.log(`${oneRecipe.title}`);
+})
```

Comment:
Here you manage to insert all the recipes to your database because you are using the insertMany method correctly. Only your console.log is not working : that is because what you receive from your query isn't one Recipe, it is an array of recipes. If you want to console.log each recipe's title, you have to loop over it like that : 
```
Recipe.insertMany(data)
  .then(recipes => {
    recipes.forEach(oneRecipe => console.log(`${oneRecipe.title}`));
  })
  .catch(err => {
  console.log("recipe insert failed", err);
});
```